game/src/Game/GameSession/billboard.gdshader

```diff
@@ -0,0 +1,45 @@
+shader_type spatial;
+//depth_test_disabled along with the sorting offset of 10
+//ensures this will be drawn in front of other objects in 
+//3d space.
+render_mode unshaded, depth_test_disabled;
+
+
+//vic2 only ever loads a max of 12 BillboardType
+//Of these, only 4 are actually used
+uniform sampler2D billboards[12] : source_color;
+uniform int numframes[12];
+uniform float sizes[12];
+
+uniform float height_add_factor = 0.3;
+
+//COLOR/INSTANCE_CUSTOM is our custom data, used as follows:
+// x=image index
+// y=frame in image index
+// z,w are unused for now
+
+void vertex() {
+	COLOR = INSTANCE_CUSTOM; //send instance_custom info to fragment
+	float size = sizes[int(COLOR.x)];
+	
+	vec3 cam_right_worldspace = vec3(VIEW_MATRIX[0][0],VIEW_MATRIX[1][0],VIEW_MATRIX[2][0]);
+	vec3 cam_up_worldspace = vec3(VIEW_MATRIX[0][1],VIEW_MATRIX[1][1],VIEW_MATRIX[2][1]);
+	VERTEX =
+		cam_right_worldspace * VERTEX.x +
+		cam_up_worldspace * VERTEX.y;
+	VERTEX.y += height_add_factor;
+	VERTEX *= size;
+}
+
+void fragment() {
+	int image_index = int(COLOR.x);
```

Comment:
```suggestion
	int image_index = int(COLOR.x + 0.5);
```
Adding `0.5` fixes the z-fighting issue on my Nvidia GPU

Reply:
that's probably down to the GPU's floating point unit performing truncation differently.

Reply:
Works.